lib/pages/feed/AuthorInsights.dart

```diff
@@ -46,17 +46,13 @@ class AuthorInsights extends StatelessWidget {
           Row(children: [
             Container(
               margin: EdgeInsets.all(5),
-              child: Column(children: [
-                commentAsset,
-                Text(commentsCount.toString())
-              ]),
+              child: Column(
+                  children: [commentAsset, Text(commentsCount.toString())]),
```

Comment:
Prefer always adding commas after parentheses, brackets etc so the text editor knows how to format your code.

Reply:
:+1:
Done